package.json

```diff
@@ -10,8 +10,8 @@
     "license": "MIT",
     "engines": {
         "vscode": "^1.74.0",
-        "node": ">=16.6.x",
-        "npm": ">=8.11.x"
+        "node": ">=16.20.x",
```

Comment:
Upgraded node and npm version to get rid of npm warnings and also updating the extension to near latest node version, so that it is guaranteed to work with version >= 16.20.1 but not less than 16.x.x.